queries.sql

```diff
@@ -179,4 +179,11 @@ JOIN species ON animals.species_id = species.id
 WHERE vets.name = 'Maisy Smith'
 GROUP BY species.name
 ORDER BY COUNT(species.name) DESC
-FETCH FIRST 1 ROW ONLY;
\ No newline at end of file
+FETCH FIRST 1 ROW ONLY;
+
+
+-- Performance
+
+EXPLAIN ANALYSE SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYSE SELECT * FROM visits where vet_id = 2;
+EXPLAIN ANALYSE SELECT * FROM owners where email = 'owner_18327@mail.com';
```

Comment:
- [ ] Based on the requirements, the execution time for these SQL queries should be `>= 1000ms`. However, I noticed that the execution time for this query is below `1000ms`:

```SQL
EXPLAIN ANALYSE SELECT * FROM owners where email = 'owner_18327@mail.com';
```
I would advise you to run this query two or three more times until the execution time is `>= 1000ms`. 

```SQL
insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
```

#### Hint: Kindly refer to this [project requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md) 👍 😎

Reply:
Thanks a lot!